File: Algorithms/01-Warmup/07-min-max-sum/07.php
<?php

$handle = fopen ("php://stdin", "r");

function miniMaxSum($arr) {
    $lower = 0;
    $higher = 0;

    for($i = 0; $i < count($arr); $i++) {
        $sum = 0;
        for($j = 0; $j < count($arr); $j++) {
            if ($j != $i) {
                $sum += $arr[$j];
            }
        }
        if ($i === 0 || $lower > $sum) {
            $lower = $sum;
        }
        if ($sum > $higher) {
            $higher = $sum;
        }
    }
    echo $lower . " " . $higher;
}

$arr_temp = fgets($handle);
$arr = explode(" ",$arr_temp);
$arr = array_map('intval', $arr);
miniMaxSum($arr);

?>

File: Algorithms/01-Warmup/07-min-max-sum/07.cpp
#include <iostream>
#include <vector>

void miniMaxSum(std::vector <int> arr) {
    long higher = 0;
    long lower = 0;
    for (int i = 0; i < arr.size(); i++) {
        long sum = 0;

        for (int j = 0; j < arr.size(); j++) {
            if (j != i) {
                sum += arr[j];
            }
        }
        if (i == 0 || lower > sum) {
            lower = sum;
        }
        if (sum > higher) {
            higher = sum;
        }
    }
    std::cout << lower << " " << higher << std::endl;
}

int main() {
    std::vector<int> arr(5);
    for(int arr_i = 0; arr_i < 5; arr_i++){
        std::cin >> arr[arr_i];
    }
    miniMaxSum(arr);
    return 0;
}
